setup.cfg

```diff
@@ -72,6 +72,7 @@ development =
     mock
     pytz
     numpy
+    shapely
```

Comment:
Adding a dev dependency on `shapely` to create the geos for the test.  Perhaps we can just use a byte string to represent the geos instead?